.dwollaci.yml

```diff
@@ -62,3 +62,34 @@ stages:
         npm install -g npm
         npm install -g serverless
         ENVIRONMENT=prod SKRIPTS_DEPLOYMENT_BUCKET=dwolla-encrypted npm run deploy
+
+additionalJobs:
+  - name: UpdateConsumerWebhookHandler
+    nodeLabel: nvm-deployer
+    parameters:
+      - name: Environment
+        type: Choice
+        description: Dwolla environments
+        choices:
+          - devint
+          - uat
+          - prod
+      - name: NodeVersion
+        type: Choice
+        description: supported node versions
+        choices:
+          - nodejs16.x
+          - nodejs20.x
+      - name: CodeName
+        type: String
+        description: Code name to use in handler
```

Comment:
will this be obvious to people or is there an example we can provide or some source that it refers to?

Reply:
Added to the description the location of the S3 bucket.